types/schemas/digitalPlanningApplication/index.ts

```diff
@@ -0,0 +1,29 @@
+import {File} from './File';
+import {Metadata} from './Metadata';
+import {PreAssessment} from './PreAssessment';
+import {Responses} from './Responses';
+import {Applicant} from './data/Applicant';
+import {Application} from './data/Application';
+import {FilesAsData} from './data/Files';
+import {Property} from './data/Property';
+import {Proposal} from './data/Proposal';
+import {User} from './data/User';
+
+/**
+ * @title Digital Planning Application
+ * @description The root specification for a planning application in England generated by a digital planning service
+ */
+export interface DigitalPlanningApplication {
```

Comment:
Something to consider - is `DigitalPlanningApplication` actually the right name? Is the `DigitalPlanning` prefix redundant given the context, will all schemas end up with the prefix?

Reply:
Very on board with simplifying at this stage :zap: agree dropping prefix and keeping `Application` only is less redundant and going to be easier to extend (eg `PreApplication`, `Assessment`, `Consultation`, `SiteNotice` etc)

Reply:
Agreed! Will pick this up in a follow up PR shortly.